examples/search.rs

```diff
@@ -54,17 +52,17 @@ async fn main() -> Result<()> {
 
     let points = vec![PointStruct::new(0, vec![12.; 10], payload)];
     client
-        .upsert_points_blocking(collection_name, None, points, None)
+        .upsert_points(UpsertPointsBuilder::new(collection_name, points))
         .await?;
 
-    let search_point_req = SearchPointsBuilder::new(collection_name, [11.; 10], 10)
-        .filter(Filter::all([Condition::matches("bar", 12)]))
-        .with_payload(PayloadIncludeSelector { fields: vec![] })
-        .params(SearchParamsBuilder::default().exact(true))
-        .build();
-
-    let search_result = client.search_points(&search_point_req).await?;
-
+    let search_result = client
+        .search_points(
+            SearchPointsBuilder::new(collection_name, [11.; 10], 10)
+                .filter(Filter::all([Condition::matches("bar", 12)]))
+                .with_payload(true)
+                .params(SearchParamsBuilder::default().exact(true)),
+        )
+        .await?;
     dbg!(&search_result);
```

Comment:
Might be nice to show conversion to json here, will also give is smaller comment

Reply:
Done, now showing it like this:

```rust
    // search_result = [
    //   {
    //     "id": 0,
    //     "version": 0,
    //     "score": 1.0000001,
    //     "payload": {
    //       "bar": 12,
    //       "baz": {
    //         "qux": "quux"
    //       },
    //       "foo": "Bar"
    //     }
    //   }
    // ]
```